src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  let discount = 0;
+
```

Comment:
Keep in mind that constants should be uppercased and be initialized on the global level, at the top of the file
```suggestion
const DAILY_PRICE = 40;
  
function calculateRentalCost(days) {
  let discount = 0;

```